Branch-SDK/src/io/branch/referral/RemoteInterface.java

```diff
@@ -156,6 +158,16 @@ private ServerResponse make_restful_get(String baseUrl, String tag, int timeout,
             timeout = DEFAULT_TIMEOUT;
         }
         if (addCommonParams(getParameters, retryNumber)) {
+            if (params != null) {
+                Iterator keys = params.keys();
+                while (keys.hasNext()) {
```

Comment:
Good catch.  Converted `addGetParams` in the `ServerRequest` class is converted  to `protected`. So only get params can be modifed inside the class .